src/components/Table/TableBody/RowActionsCell/_row-actions-cell.scss

```diff
@@ -0,0 +1,8 @@
+@import '../../../../globals/vars';
+
+.#{$iot-prefix}--btn--icononly {
```

Comment:
Would be best to scope this to a block level identifier of the component

```suggestion
.#{$iot-prefix}--row-actions-cell-btn--icononly {
```

Reply:
Makes sense, thanks!